### DISCUSSION TITLE
Place index file in a different output location

### DISCUSSION THREAD

**Discussion by Aristidis10** [Doraku/DefaultDocumentation#144] [Q&A]
I am trying to get the documentation to work in Azure Wiki. I created a separate output folder for each project in my solution for a .NET app. I then put al those folders in another called "docs" where I use as the wiki for the entire solution. I was hoping to output the rest of the documentation in the specific project folders, but having the assembly file (index.md) in the "docs" folder. This would allow Azure to show the index when clicking on the page content of the folder instead of having to search for it. Is there a way to output the assembly page to a different location?

**Comment by Doraku** [comment]
Pretty late reply but since you have multiple projects/assemblies won't you end up with multiple index.md files? Or I'm missing something.

**Comment by Doraku** [comment]
Reading old issue/discussion, I think I now get what you. You could achieve this by creating a [plugin](https://github.com/Doraku/DefaultDocumentation?tab=readme-ov-file#Configuration_Plugins) that specifically handle the file path of the assembly doc page. I'll close this but feel free to ask if you need more inputs.
